frontend/Props.js

```diff
@@ -16,10 +16,7 @@ var PropVal = require('./PropVal');
 class Props extends React.Component {
   props: Object;
   shouldComponentUpdate(nextProps: Object): boolean {
-    if (nextProps === this.props) {
-      return false;
-    }
-    return true;
+    return nextProps !== this.props;
```

Comment:
I was a bit confused myself but since I'm new to the codebase I've decided not to change the logic beneath it, only polish the code a little